### PR TITLE
Add __init__.py to fuzzbench

### PR DESCRIPTION
**Pull request by zchcai**

### CODE REVIEW

**Comment by jonathanmetzman** [comment]
Does this fix the issue? Please link

**Comment by zchcai** [comment]
This PR may fix #555. Under testing now.

**Comment by jonathanmetzman** [comment]
TO test you might need to make a dummy change to the file that failed linting since we only lint changed files (which is incorrect behavior, but a different issue).

**Comment by zchcai** [comment]
> TO test you might need to make a dummy change to the file that failed linting since we only lint changed files (which is incorrect behavior, but a different issue).

Oh, I see. Let me try again.

**Comment by zchcai** [comment]
> TO test you might need to make a dummy change to the file that failed linting since we only lint changed files (which is incorrect behavior, but a different issue).

Well, adding `__init__.py` under fuzzbench/fuzzbench does not help. I check this by modifying a comment in fuzzbench/run_experiment.py:
```
source .venv/bin/activate && python3 presubmit.py lint
************* Module fuzzbench.fuzzbench.run_experiment
fuzzbench/run_experiment.py:21:0: E0611: No name 'fake_jobs' in module 'fuzzbench' (no-name-in-module)

-----------------------------------
Your code has been rated at 7.92/10

ERROR: lint failed, see errors above.
Failed.
make: *** [Makefile:60: lint] Error 1
```

**Comment by zchcai** [comment]
As we can see, the lint check fails now.

**Comment by zchcai** [comment]
By the way, despite #555, shall we add this `__init__.py` here?

**Comment by jonathanmetzman** [comment]
Sure. It doesn't really matter to me.

**Comment by lszekeres** [comment]
> By the way, despite #555, shall we add this `__init__.py` here?

I think empty __init__.py files are not necessary from python 3.3. why add them if we don't need them?

**Comment by jonathanmetzman** [comment]
> > By the way, despite #555, shall we add this `__init__.py` here?
> 
> I think empty `__init__.py` files are not necessary from python 3.3. why add them if we don't need them?

I've definitely seen cases where not having them has been an issue. in src_analysis I had to write my own tooling because Python's `modulefinder` library. And I feel like I've seen cases where not having them messes up linting although I can't say for sure this is the case. At this point I think it's better to have them since there is no cost to doing so, but not having them can have a cost.

**Comment by zchcai** [comment]
According to #566, this PR should be closed now to fix #555
